### PR TITLE
New Training: Introduction to Data Management Plan (DMP) for Peatland Research and PeatDataHub

### PR DESCRIPTION
**Pull request by kkamieniecka**
Peatland monitoring and research generates data that is used for management, conservation and policy related to these ecosystems. To ensure that your project data is collected systematically and can be analysed, shared, and re-used it is important to have a data management plan.

This tutorial is a short introduction to writing a Data Management Plan (DMP) with examples for peatland monitoring projects and initiatives.

### CODE REVIEW

File: topics/fair/tutorials/data-management-peatlands/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+level: Introductory
+abbreviations:
+  
+  DMP: Data Management Plan
+  FAIR: Findable, Accessible, Interoperable, Reusable
+zenodo_link: ''
+questions:
+- Why create a DMP for peatland research?
+- How to apply FAIR data management solutions for peatland research?
+Objectives:
```
Comment (shiltemann) [inline topics/fair/tutorials/data-management-peatlands/tutorial.md:13]:
```suggestion
objectives:
```

File: topics/fair/tutorials/data-management-peatlands/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+* Reusable (R): Storage and backup supported by legal and ethical requirements, as well as data release policies.
+
+More information is available in the [FAIR data management tutorial](https://training.galaxyproject.org/training-material/topics/fair/tutorials/data-management/tutorial.html)
+
+# Conclusion
+
+Creating a DMP will allow you and your institution to make the most of your peatland data. This introduction to DMP aims to highlight the usefulness of data management for peatland research and monitoring.
+
+# References
+*Wilkinson, M. D., M. Dumontier, I. J. J. Aalbersberg, G. Appleton, M. Axton et al., 2016 The FAIR Guiding Principles for scientific data management and stewardship. Scientific data 3: 1–9. [10.1038/sdata.2016.18](https://www.nature.com/articles/sdata201618)
```
Comment (shiltemann) [inline topics/fair/tutorials/data-management-peatlands/tutorial.md:103]: Could you create a tutorial.bib file with this citation? The References section will then be automatically added to the end of the tutorial.

File: topics/fair/tutorials/data-management-peatlands/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    - angiolini
+    - poterlowicz-lab
+  editing:
+    - hexylena
+subtopic: fair-data
+
+---
+
+
+# Introduction
```
Comment (shiltemann) [inline topics/fair/tutorials/data-management-peatlands/tutorial.md:41]: We now try to remove this Introduction header, makes it easier to generate informative preview links on e.g. social media
```suggestion

```

**Comment by kkamieniecka** [comment]
Hi @shiltemann thank you for your review. I think we fixed everything now

**Comment by shiltemann** [comment]
Hi @kkamieniecka thanks a lot! Did you perhaps forget to push the `tutorial.bib` file? I tried to do it but I don't have permission to push to the elixir-uk fork.

Could you either add me as a collaborator on your fork, or add this file yourself, with the following content (generated using doi2bib). Then we can merge :) Thanks!

```
@article{Wilkinson2016,
  title = {The FAIR Guiding Principles for scientific data management and stewardship},
  volume = {3},
  ISSN = {2052-4463},
  url = {http://dx.doi.org/10.1038/sdata.2016.18},
  DOI = {10.1038/sdata.2016.18},
  number = {1},
  journal = {Scientific Data},
  publisher = {Springer Science and Business Media LLC},
  author = {Wilkinson,  Mark D. and Dumontier,  Michel and Aalbersberg,  IJsbrand Jan and Appleton,  Gabrielle and Axton,  Myles and Baak,  Arie and Blomberg,  Niklas and Boiten,  Jan-Willem and da Silva Santos,  Luiz Bonino and Bourne,  Philip E. and Bouwman,  Jildau and Brookes,  Anthony J. and Clark,  Tim and Crosas,  Mercè and Dillo,  Ingrid and Dumon,  Olivier and Edmunds,  Scott and Evelo,  Chris T. and Finkers,  Richard and Gonzalez-Beltran,  Alejandra and Gray,  Alasdair J.G. and Groth,  Paul and Goble,  Carole and Grethe,  Jeffrey S. and Heringa,  Jaap and ’t Hoen,  Peter A.C and Hooft,  Rob and Kuhn,  Tobias and Kok,  Ruben and Kok,  Joost and Lusher,  Scott J. and Martone,  Maryann E. and Mons,  Albert and Packer,  Abel L. and Persson,  Bengt and Rocca-Serra,  Philippe and Roos,  Marco and van Schaik,  Rene and Sansone,  Susanna-Assunta and Schultes,  Erik and Sengstag,  Thierry and Slater,  Ted and Strawn,  George and Swertz,  Morris A. and Thompson,  Mark and van der Lei,  Johan and van Mulligen,  Erik and Velterop,  Jan and Waagmeester,  Andra and Wittenburg,  Peter and Wolstencroft,  Katherine and Zhao,  Jun and Mons,  Barend},
  year = {2016},
  month = mar 
}
```

File: topics/fair/tutorials/data-management-peatlands/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+|Capabilities| &bull; Evaluation of institutional capabilities to preserve and manage data|
+|Maintaining and implementing the Data Management Plan|&bull; Actions to maintain and deliver the DMP |
+|Environmental considerations|&bull; Energy cost considerations related to data management|
+|Responsibilities|&bull; Agreement on the responsibilities of named individuals for data management|
+|Relevant institutional, departmental or study policies on data sharing and data security| &bull; Review of institutional policies for data management|
+
+## Hands-on: Create a DMP for peatland research
+* Download the DMP template for [Peatland Research and Monitoring](https://zenodo.org/records/11185774).
+* Fill in the DMP template using the prompt questions.
+* Review your DMP with your team and collaborators.
```
Comment (hexylena) [inline topics/fair/tutorials/data-management-peatlands/tutorial.md:80]: Would it be possible to include a completed exemplar here?

Comment (kkamieniecka) [reply topics/fair/tutorials/data-management-peatlands/tutorial.md:80]: Hi Helena, sorry I've missed your comment. We will add an example and attach supplementary video after workshop on Thursday.

Comment (kkamieniecka) [reply topics/fair/tutorials/data-management-peatlands/tutorial.md:80]: @hexylena @shiltemann Hi both, sorry to bother you can we merge it this way now so we can update after workshop and still distribute the lesson URL? Thank you!

File: topics/fair/tutorials/data-management-peatlands/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  authorship:
+    - GabrielaLopez
+    - kkamieniecka
+    - robertmand
+    - angiolini
+    - poterlowicz-lab
+  editing:
+    - hexylena
+subtopic: fair-data
+
```
Comment (khaled196) [inline topics/fair/tutorials/data-management-peatlands/tutorial.md:36]:
```suggestion

recordings:
  - speakers:
    - GabrielaLopez
    captioners:
    - kkamieniecka
    date: '2024-09-12'
    length: 1H1M41S
    youtube_id: "MX-4AWyKP8E"

```

**Review by khaled196 — CHANGES_REQUESTED** [review]

File: topics/fair/tutorials/data-management-peatlands/tutorial.md (hunk 4 above)
Comment (kkamieniecka) [reply topics/fair/tutorials/data-management-peatlands/tutorial.md:89]: Hi @hexylena we have now example document on zenodo and video of the interactive training session, please merge

Comment (hexylena) [reply topics/fair/tutorials/data-management-peatlands/tutorial.md:89]: hi @kkamieniecka apologies for missing the jun6 message. Yes of course!

**Review by hexylena — APPROVED** [review]

**Pull request merged**